ramp-database/ramp_database/model/submission.py

```diff
@@ -408,6 +408,7 @@ def set_state(self, state, session=None):
             all_cv_folds = (session.query(SubmissionOnCVFold)
                                    .filter_by(submission_id=self.id)
                                    .all())
+            all_cv_folds = sorted(all_cv_folds, key=lambda x: x.id)
```

Comment:
Thanks! I think,
```py
  .filter_by(submission_id=self.id)
  .order_by(SubmissionOnCVFold.id)
  .all()
```
would be a more efficient way of doing this as sorting is then done at the DB level.

So there shouldn't be issues with backward compatibility here, right?